src/components/ui/Panel.jsx

```diff
@@ -119,10 +121,13 @@ export default class Panel extends React.Component {
     this.removeListeners();
 
     this.moveHandler = event => this.move(event, forceResize);
+    this.moveTarget = event.target;
+    // It is important to attach the move handler to the current target (instead of `document`)
+    // in order to keep receiving the events, even if the target is removed from the DOM.
```

Comment:
I don't think this solves anything. In both cases the target element is the same, and if it suddenly disappears from the DOM it stops bubbling `touch/mousemove` events. The element where the listener is attached doesn't matter here.

Reply:
By attaching the `touchmove` event handler to the target of the `touchstart` event, events no longer need to bubble up as the listener function is called directly. At least, that's what I had observed on both Chrome and Firefox on Android, and I could't reproduce the bug after this change.

But I suppose I may have missed something about the conditions when the bug appears :shrug: